soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -1125,11 +1125,19 @@ void DrawEquipmentTab() {
     DrawUpgradeIcon("Strength", UPG_STRENGTH, strengthValues);
 
     // There is no icon for child wallet, so default to a text list
-    const std::vector<std::string> walletNames = {
+    // this was const, but I needed to append to it depending in rando settings.
+    std::vector<std::string> walletNamesImpl = {
```

Comment:
I wasn't really sure what to call it, but in case there was a reason that array was const before, I made a new array that could be one thing or the other depending on those settings. I considered that the "Implementation" of that array and then assigned the end result to the original const variable. I'm definitely down to change the name if needed.